uk_bin_collection/uk_bin_collection/councils/MidSuffolkDistrictCouncil.py

```
@@ -1,13 +1,11 @@
-import re
+import datetime
 import time
+from datetime import datetime
```

Ruff (0.14.14)

[error] Redefinition of unused `datetime` from line 1: `datetime` redefined here

(F811)